src/ray/gcs/gcs_server/gcs_actor_scheduler.cc

```diff
@@ -440,6 +440,18 @@ void RayletBasedActorScheduler::HandleWorkerLeaseReply(
   // gcs_actor_manager will reconstruct it again.
   auto node_id = NodeID::FromBinary(node->node_id());
   auto iter = node_to_actors_when_leasing_.find(node_id);
+  auto kill_worker = [&reply, this]() {
```

Comment:
Why don't we make it as a private method?

Reply:
I think this kill_worker should be scoped to this function only. The reason is that it's actually coupling with this one: it set the actor id = Nil. It means the worker is leased, but not assigned any actor.

In actor_manager, there actually is a function to kill an actor.

Reply:
That makes sense. Can you comment this here (so we can avoid confusion in the future)?